### PR TITLE
Create Rule21-18.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

**Review by supriyagoel — APPROVED** [review]

File: docs/section21/Rule21-18.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Case 1: If the total number of heating hot water loop in B_RMR is 1: `if hhw_loop_count == 1: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Note:**
+
+1. I was not able to find the exact language in App. G to put under the "Appendix G Section Reference".
+2. Is this requirement for B_RMR or for each building under B_RMR?
```
Comment (mkarpman) [inline docs/section21/Rule21-18.md:41]: This is for the entire B-RMR, i.e., all buildings included in the B-RMR. I beleive your RDS reflects that.

File: docs/section21/Rule21-18.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- Case 1: If the total number of heating hot water loop in B_RMR is 1: `if hhw_loop_count == 1: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Note:**
+
+1. I was not able to find the exact language in App. G to put under the "Appendix G Section Reference".
```
Comment (mkarpman) [inline docs/section21/Rule21-18.md:40]: I don't think there is an explicit language requiring that, but it is implied.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

File: docs/section21/Rule21-18.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+**Rule ID:** 21-18  
+**Rule Description:** Baseline shall have only one heating hot water plant.  
+**Rule Assertion:** B-RMR LEN(FluidLoop.type == "HEATING") == 1  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. P-RMR is not modeled with purchased heating.
```
Comment (mkarpman) [inline docs/section21/Rule21-18.md:13]: I think this rule should apply to projects with purchased heating as well.

File: docs/section21/Rule21-18.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
+  - Else, P-RMR is modeled with purchased cooling: `else:`
+
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
```
Comment (mkarpman) [inline docs/section21/Rule21-18.md:44]: Suggest not differentiating between projects with purchased and on-site cooling.

File: docs/section21/Rule21-18.md (hunk 3 above)
Comment (supriyagoel) [reply docs/section21/Rule21-18.md:13]: as discussed RDS will be updated to include PHW, b systems and c systems will be added to the applicability check

File: docs/section21/Rule21-18.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+**Rule Description:** Baseline shall have only one heating hot water plant.  
+**Rule Assertion:** B-RMR LEN(FluidLoop.type == "HEATING") == 1  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11(for climate zones other than 0 through 3A), 12, 1a, 7a, 11a(for climate zones other than 0 through 3A), 12a.
+
```
Comment (supriyagoel) [inline docs/section21/Rule21-18.md:14]: based on the revised logic which includes PHW plans, we should probably add systems 1B, 3B, 5B, 6B, 7B, 8B, 9B, 11B, 12B, 13B, 1C, 3C, 7C, 11C, 12C, 13C to the list. @mkarpman please confirm.

**Review by supriyagoel — CHANGES_REQUESTED** [review]

File: docs/section21/Rule21-18.md (hunk 5 above)
Comment (mkarpman) [reply docs/section21/Rule21-18.md:14]: Yes, I agree.

Comment (supriyagoel) [reply docs/section21/Rule21-18.md:14]: need to add PHW systems (c series)

**Review by supriyagoel — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**